main.variant

```diff
@@ -39,6 +112,32 @@ job "generate" {
 		short = "a"
 	}
 
+	# ignored
```

Comment:
I preferred to have these ignored flags because no one is likely to look at the flags for this variant command. In the end, [github docker actions](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-docker-actions) doesn't allow an action to have multiple entrypoints or other fancy stuff, afaik. So instead, I just allow either set of flags and the 'type' will pick which job from variant to run. That in turn picks what to ignore or allow.